spateo/plotting/static/pairwise.py

```diff
@@ -0,0 +1,185 @@
+"""
+Plots for pairwise comparison analyses between groups of samples and across different conditions (e.g. biological
+conditions or using different clustering parameters).
+
+These functions build on the Label class- see spateo.utils.Label for documentation
+"""
+from typing import List, Tuple, Union
+
+import matplotlib as mpl
+import matplotlib.patheffects as PathEffects
+import matplotlib.pyplot as plt
+import numpy as np
+import scipy
+from anndata import AnnData
+from matplotlib.collections import PolyCollection
+from matplotlib.ticker import StrMethodFormatter
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+
+from ...utils import Label, interlabel_connections
+
+plt.rcParams["figure.dpi"] = 300
+
+
+def plot_connections(
+    label: Label,
+    weights_matrix: Union[scipy.sparse.csr_matrix, np.ndarray],
+    ax: Union[None, mpl.axes.Axes] = None,
+    figsize: tuple = (8, 8),
+    zero_self_connections: bool = True,
+    normalize_by_self_connections: bool = False,
+    shapes_style: bool = True,
+    max_scale: float = 0.46,
+    colormap: Union[str, dict, "mpl.colormap"] = "Spectral",
+    title_str="connection strengths between types",
+    title_fontsize: float = 12,
+    label_fontsize: float = 12,
+    verbose: bool = True,
+) -> None:
+    """
+    Plot connections between labels- visualization of how closely labels are colocalized
+
+    Parameters
+    ----------
+    shapes_style : bool, default True
+        If True plots squares, if False plots heatmap
+    max_scale : float, default 0.46
+        Only used for the case that 'shape_style' is True, gives maximum size of square
+    colormap : str, dict, or matplotlib.colormap
+        Specifies colors to use for plotting. If dictionary, keys should be numerical labels corresponding to those
+        of the Label object.
+    """
+    if ax is None:
+        fig, ax = plt.subplots(figsize=figsize)
+    else:
+        ax = ax
+        fig = ax.get_figure()
+    connections = interlabel_connections(label, weights_matrix)
+
+    if zero_self_connections:
+        np.fill_diagonal(connections, 0)
+    elif normalize_by_self_connections:
+        connections /= connections.diagonal()[:, np.newaxis]
+
+    connections_max = np.amax(connections)
+
+    # Set label colors:
+    if isinstance(colormap, str):
+        cmap = mpl.cm.get_cmap(colormap)
+    else:
+        cmap = colormap
+
+    # If colormap is given, map label ID to points along the colormap. If dictionary is given, instead map
```

Comment:
instead map `label ID` to the color names? 

need to complete the above sentence